scripts/check-chart-kdm-source-values

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Without RELEASE_TYPE set, it will show the values for each.
+# With RELEASE_TYPE set to 'rc' or 'final-rc', it will compare the values to the expected ones and report CORRECT or INCORRECT
+
+case $RELEASE_TYPE in
+  rc)
+    REQUIRED_VALUE="dev-v2.6"
+    ;;
+  final-rc)
+    REQUIRED_VALUE="release-v2.6"
+    ;;
+esac
+
+if [ "x${RELEASE_TYPE}" != "x" ]; then
```

Comment:
Could we use `-z $RELEASE_TYPE` instead?

Reply:
Changed it to `-n`